### PR TITLE
Use Plone minor version in classifiers, and various other small fixes

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Too small for separate pull requests, and they all look safe to me, but this is my first contribution to bobtemplates.plone, so someone might as well have a look at the bunch.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
Travis does not like it, and it has a point. Apparently there is a bin/nosetests that I should have run. Don't tell me I needed to read some documentation... ;-)

File: nosetests_answers.ini [hunk 1]
```diff
@@ -15,4 +15,5 @@ author.irc = irc.freenode.org#plone
 
 plone.version = 4.3.6
 plone.is_plone5 = False
+plone.minor_version = 4.3
```
Comment (tisto) [inline nosetests_answers.ini]: I think we can extract the minor version from plone.version without adding an additional question.

**Comment by tisto** [comment]
I guess we should create a wrapper "test" script that runs all acceptance tests and nosetests.

File: nosetests_answers.ini (hunk 1 above)
Comment (mauritsvanrees) [inline nosetests_answers.ini]: We *are* extracting it from plone.version. But I guess when you use a .ini file, the code that extracts it (`post_plone_version` in `hooks.py`) is not getting called.

**Comment by mauritsvanrees** [comment]
I need to revisit this for a minor update once pull request #101 is merged.

**Comment by mauritsvanrees** [comment]
I rebased. Tests now pass. Coverage is not working yet because that is in a different pull request.
Ready for review.

**Pull request merged**